doc/manual/src/hacking.md

```diff
@@ -44,3 +43,15 @@ in the root of the repository at least once.
 if run with high parallelism [due to an issue in
 `Test::PostgreSQL`](https://github.com/TJC/Test-postgresql/issues/40)
 causing database ports to collide.
+
+## Working on the Manual
+
+By default, `foreman start` runs mdbook in "watch" mode. mdbook listens
+at [http://localhost:63332/](http://localhost:63332/) by default, and
+will reload the page every time you save.
+
+## Building
+
+To build Hydra and its dependencies:
+
+    $ nix-build release.nix -A build.x86_64-linux
```

Comment:
Are there other platforms a user might want to know about?

Reply:
None that I know of where people actually deploy Hydra today...